scripts/class/Hero.js

```diff
@@ -5,6 +5,20 @@ class Hero {
     */
     constructor(level){
         //TODO generate initial hero/random monster with random stats
+        this.name = '';
+        this.experience = 0;
+        this.level = level;
```

Comment:
`level || 1`, because the default behavior for creating a new hero should be a hero with level 1